Neos.RedirectHandler/Classes/Aspect/RedirectionStorageAspect.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Neos\RedirectHandler\Aspect;
+
+/*
+ * This file is part of the Neos.RedirectHandler package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\RedirectHandler\DatabaseStorage\Domain\Model\Redirection;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Aop\JoinPointInterface;
+use TYPO3\Flow\Log\SystemLoggerInterface;
+use \Neos\RedirectHandler\Redirection as RedirectionDto;
+
+/**
+ * Redirection Storage Aspect
+ *
+ * This aspect is responsible to emit a signal just after a redirection has been added.
```

Comment:
@bwaidelich  The RedirectionService is used to generation the redirection, not for storage management. So use can inject the StorageInterface, and use the Storage directly. This aspect simplify additional storage coding and enforce the call of the signal emitter